db/schema.rb

```diff
@@ -55,6 +55,33 @@
   add_index "identities", ["session_uuid"], name: "index_identities_on_session_uuid", unique: true, using: :btree
   add_index "identities", ["user_id"], name: "index_identities_on_user_id", using: :btree
 
+  create_table "piis", force: :cascade do |t|
```

Comment:
Sorry, I know this change will require a lot of changes, but I don't think we should call this page piis.   We also store PII in the user table so its misleading.  Also this is specific type of PII.. that is verified/proofed data.

How about "proofing_records", "verified_attributes"?... or something better  I can't think of right now? :)